packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/loadingaconfiguration.md

```diff
@@ -0,0 +1,67 @@
+Initialize a configuration by passing the appropriate values to
+`IDXClient.Configuration(issuer:clientId:clientSecret:scopes:redirectUri:)`. You can set
+the values directly in your code or load them from a file. This example shows loading the
+values from a property file in your project.
+
+First, create a property list file, such as `Okta.plist`. Next, add key-value pairs with
```

Comment:
Nice cach (😉), how about this modification:

> ...Application Integration. For example, in the following text version of a `plist`, substitute...

You are correct in that not all devs realize a plist is just an XML file in a particular format, however since the XML explicitly contains a `DOCTYPE` statement identifying it as a `plist`, I think that's OK.

Reply:
Thanks for the 🍵 😉.

I modified the text slightly to be clear in an XML file of a plist (which should be clear as there's a `DOCTYPE` header denoting `plist`. The text now reads:

> ...Application Integration. For example, in the following text version of a `plist`, substitute your issuer URL...

Reply:
No problem.  "The xyz below" hangup is a killer in books. Frequently layout moves the xyz somewhere other than "below" :-)

Your suggestion reads fine to me. Application Integration still goes to app integration though.